doc/faq.rst

```diff
@@ -300,6 +300,30 @@ reviewers are busy. We ask for your understanding and request that you
 not close your pull request or discontinue your work solely because of
 this reason.
 
+What does the "spam" label for issues or pull requests mean?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The "spam" label is an indication for reviewers that the issue or
+pull request may not have received sufficient effort or preparation
+from the author for a productive review. The maintainers are using this label
+as a way to deal with the increase of low quality PRs and issues.
```

Comment:
Maybe the problem should better be phrased in terms of value rather than quality. We are not interested in high quality solutions to problems nobody has faced or expressed in the past.

```suggestion
as a way to deal with the increase of low value PRs and issues.
```